README.md

```diff
@@ -1,2 +1,39 @@
 # openjdk9-openj9-releases
 Eclipse openj9 release repo
+
+This repository is for publishing releases of OpenJDK with OpenJ9 (an enhanced JVM replacement for HotSpot).
+(http://openjdk.java.net)
+(http://https://github.com/eclipse/openj9)
+
+These binaries are built on a shared CI infrastructure (see http://ci.adoptopenjdk.net) using an openly shared build tool chain (see https://github.com/AdoptOpenJDK/openjdk-build)
+
+## WARNING: THESE ARE NOT FOR PRODUCTION USE
+
+The AdoptOpenJDK OpenJDK binaries are not yet fully tested (JCK/TCK + further testing) and verified. Currently for production you should look to professionally supported Java implementations from Oracle, Azul, IBM, RedHat, SAP and other vendors.
+
+## Download OpenJDK Release binaries
+
+Find all the prebuilt AdoptOpenJDK OpenJDK with OpenJ9 release binaries [here](https://github.com/AdoptOpenJDK/openjdk9-openj9-releases)
```

Comment:
Do you mean this hyperlink to go to the github repo, or rather the download site (https://adoptopenjdk.net/archive.html?variant=openjdk9-openj9) ?

Reply:
yes - good spot - link corrected